### ISSUE TITLE
Ability to erase or not save audio files to a .integra file

### ISSUE THREAD

**Issue by jrdooley** [BirminghamConservatoire/IntegraLive#938]
I am trying to use the Soundfiler module as a looper. I am recording live audio and looping it, but do not wish to save it with the project file. Currently, Integra saves this audio with my project, meaning that when I re-open the project my old samples are playing back on loop. This makes the project file unnecessarily big and requires me to reset the samplers to ensure that no 'old' sounds are playing. A simple answer is not to save the .integra file after recording, though this is not ideal if I have made edits elsewhere in the project file.

**Comment by balandinodidonato** [comment]
Hi there @jrdooley, did you implement the *clear* button?

**Comment by jrdooley** [comment]
No, I did not implement the 'clear' button idea. I do think this would be a very useful feature, though.
